cdap-ui/app/features/overview/overview.html

```diff
@@ -101,29 +66,30 @@ <h3 class="text-uppercase"> Accelerate... </h3>
 
     <div class="col-xs-12 col-lg-8">
```

Comment:
For now this is not done. Moving this to another PR since the layout becomes un-necessarily complicated to be addressed in one PR.